### ISSUE TITLE
Missing variable in _variables.scss

### ISSUE THREAD

**Issue by pioz** [andrewgioia/mana#76]
In the file `_variables.scss` the variable `$ms-ability-grad` used in file `_extras.scss` not exists, so when I import the mana.scss file I get an error.

I think you need to add
```
$ms-ability-grad: linear-gradient(0deg, #c8913b 0%, #ecc65a 100%);
```
in file `_variables.scss`.

**Comment by mcottret** [comment]
Duplicate of https://github.com/andrewgioia/mana/issues/64

**Comment by andrewgioia** [comment]
Fixed in 1.13.0
